api/scpca_portal/models/sample.py

```diff
@@ -60,11 +61,12 @@ def create_from_dict(cls, data, project):
         sample = cls(
             additional_metadata=additional_metadata,
             age_at_diagnosis=data["age"],
-            cell_count=data["cell_count"],
+            cell_count=data.get("cell_count", 0),
```

Comment:
I think we need to confirm that this should be either 0 or null where we would display 'N/A' on the client.

Reply:
We certainly do.
Would you prefer to address it in this PR or a separate one?

Reply:
Ill make an issue for this